### PR TITLE
queries/*/highlights.scm : remove @spell for strings

### PR DESCRIPTION
**Pull request by ShellCode33**
I'm not familiar with scm files syntax, review carefully :)

Related #5202

Ping @lewis6991

### CODE REVIEW

File: queries/haskell/highlights.scm [hunk 1]
```diff
@@ -156,6 +156,5 @@
 ;; ----------------------------------------------------------------------------
 ;; Spell checking
 
-(string) @spell
```
Comment (ShellCode33) [inline queries/haskell/highlights.scm:159]: Should the line be completely removed ?

File: queries/racket/highlights.scm [hunk 2]
```diff
@@ -20,7 +20,6 @@
 [(string)
  (here_string)
  (byte_string)] @string
-(string) @spell
```
Comment (ShellCode33) [inline queries/racket/highlights.scm:23]: Should the line be completely removed ?

File: queries/ruby/highlights.scm [hunk 3]
```diff
@@ -192,8 +192,6 @@
   (comment)+ @comment.documentation
   (method))
 
-(string_content) @spell
```
Comment (ShellCode33) [inline queries/ruby/highlights.scm:195]: Should the line be completely removed ?

**Comment by amaanq** [comment]
The lines can be completely removed, and you can remove extra spaces/headers from relevant queries - looks fine otherwise

**Comment by ShellCode33** [comment]
Done ! Maybe the CI should be run on this ?

**Review by amaanq — APPROVED** [review]

File: queries/json/highlights.scm [hunk 4]
```diff
@@ -12,8 +12,6 @@
 
 (array (string) @string)
 
-(string_content) @spell
```
[the remark this answers is not in the document]
Comment (lewis6991) [reply queries/json/highlights.scm:15]: Honestly, I think this one should stay.

[the remark this answers is not in the document]
Comment (clason) [reply queries/json/highlights.scm:15]: In my experience, it's mostly commit hashes...

[the remark this answers is not in the document]
Comment (lewis6991) [reply queries/json/highlights.scm:15]: Let's go with this then.

**Pull request merged**